operator/builtin/output/forward/forward.go

```diff
@@ -101,7 +103,34 @@ func (f *ForwardOutput) Stop() error {
 	f.cancel()
 	f.wg.Wait()
 	f.flusher.Stop()
-	return f.buffer.Close()
+
+	entries, err := f.buffer.Close()
+	if err != nil {
+		f.Error("Failed to retrieve entries")
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	err = f.sendEntries(ctx, entries)
```

Comment:
Looks like we aren't protecting against len(entries) == 0, so it looks like we'll send a request with 0 entries?